### ISSUE TITLE
Node 22 engines: should it use range instead?

### ISSUE THREAD

**Issue by akasection** [IlyaSemenov/nuxt-vite-legacy#17]
Hi @IlyaSemenov! forwarding from the commit comment here: https://github.com/IlyaSemenov/nuxt-vite-legacy/commit/24d68e33d65beca783b65996460f4f5c22e7174e#r148255409

Just wondering on why the node engine in `package.json` is now set to 22? it seems node 22 hasn't been LTS yet.
And since nuxt still works on node 18+, how about make it as range? (previously 20)

```
  },
  "engines": {
    "node": ">=18",
    "pnpm": ">=9"
  },
```

**Comment by IlyaSemenov** [comment]
You're right, there should be no `engines` at all as there are no strict limitations for either field imposed by the library itself.

I figured I was misusing it as I was under impression it's intended for library maintainers and not for the consumers, while in fact it's the latter.

Thanks for the heads up.

**Comment by IlyaSemenov** [comment]
I published the fix in 1.3.1.

**Comment by akasection** [comment]
Ah, great! Yep now it's working properly.

You're welcome 🫡
